### PR TITLE
Mod: update the comments in metrics

### PR DESCRIPTION
**Pull request by williamfeng323**
**What this PR does**:
Mod: Updated the comments in metrics, corrected the grammar.

### CODE REVIEW

File: metrics/prometheus/reporter.go [hunk 1]
```diff
@@ -23,19 +23,15 @@ import (
 	"strings"
 	"sync"
 	"time"
-)
-import (
```
Comment (AlexStocks) [inline metrics/prometheus/reporter.go:27]: pls reserve this import block. And do not merge it into the next block.

File: metrics/reporter.go [hunk 2]
```diff
@@ -20,16 +20,15 @@ package metrics
 import (
 	"context"
 	"time"
-)
-import (
+
```
Comment (zouyx) [inline metrics/reporter.go:23]: revert this change pls
